terraform/variables.tf

```diff
@@ -11,35 +11,71 @@ variable "mgmt_vpc_name" {
   type = "string"
 }
 variable "mgmt_vpc_cidr" {
-  type = "string"
+  default = "10.0.0.0/16"
 }
 variable "mgmt_public_subnet_cidrs" {
-  type = "list"
+  default = ["10.0.0.0/24"]
 }
 variable "mgmt_private_subnet_cidrs" {
-  type = "list"
+  default = ["10.0.1.0/24"]
 }
 variable "mgmt_database_subnet_cidrs" {
-  type = "list"
+  default = ["10.0.101.0/24", "10.0.102.0/24"]
 }
 variable "mgmt_nat_gateway" {
   type = "string"
+  default = "true"
 }
 variable "mgmt_dns_hostnames" {
   type = "string"
+  default = "true"
 }
 variable "mgmt_dns_support" {
   type = "string"
+  default = "true"
 }
 variable "private_mgmt_zone_name" {
   type = "string"
 }
 variable "mgmt_flow_log_group_name" {
+  default = "vpc_mgmt_flow_logs"
+}
+variable "app_vpc_name" {
+  type = "string"
+}
+variable "app_vpc_cidr" {
+  default = "10.1.0.0/16"
+}
+variable "app_public_subnet_cidrs" {
+  default = ["10.1.0.0/24"]
+}
+variable "app_private_subnet_cidrs" {
+  default = ["10.1.1.0/24"]
+}
+variable "app_database_subnet_cidrs" {
+  default = ["10.1.101.0/24", "10.1.102.0/24"]
+}
+variable "app_nat_gateway" {
   type = "string"
```

Comment:
FYI that [`type` is inferred when you have a `default`](https://www.terraform.io/docs/configuration/variables.html#type).